packages/sui-studio/src/components/test/index.scss

```diff
@@ -7,7 +7,8 @@
   &--open,
   &--failures {
     display: block;
-    max-width: 500px;
+    max-width: 700px;
```

Comment:
Fine tunning. XD

Reply:
😆 Time ago when I added this I would that it was okay, but the truth is, that for long tests it's not wider enough

Reply:
I **never** do so many errors to reach this height.. but if you do.. It's ok :trollface:

Reply:
well, but I guess you test a lot, don't you? :trollface:

Reply:
<img width="494" alt="Screenshot 2020-05-21 at 10 40 36" src="https://user-images.githubusercontent.com/10925540/82540575-88ea4e80-9b4f-11ea-93d7-d6363bff94a9.png">